### PR TITLE
user can delete his own comment

### PR DESCRIPTION
**Pull request by ostgit12**
## Description

Now user can delete his own comment and his own reply for any other comment

### Checklist
- [ x ] 🔽 My branch is up-to-date with "development" branch
- [ x ] ✅All tests passed locally
- [ x ] ✨My changes working with up-to-date front-end and admin part locally, like charm
- [ x ] 🔗 Link pull request to issue

### CODE REVIEW

File: modules/comment/comment.permissions.js [hunk 1]
```diff
@@ -47,8 +47,8 @@ const commentPermissionsMutations = {
     inputDataValidation(REPLY_COMMENT_DATA, replyCommentUpdateValidator),
     hasRoles([ADMIN, SUPERADMIN])
   ),
-  deleteComment: or(isTheSameUser, hasRoles([ADMIN, SUPERADMIN])),
-  deleteReplyForComment: or(isTheSameUser, hasRoles([ADMIN, SUPERADMIN])),
+  deleteComment: or(isTheSameUser, hasRoles([ADMIN, SUPERADMIN, USER])),
```
Comment (maarekua) [inline modules/comment/comment.permissions.js:50]: For me, it looks like any member with role USER can remove any comment.

**Review by maarekua — CHANGES_REQUESTED** [review]

File: modules/comment/comment.permissions.js (hunk 1 above)
Comment (AndriyLymych) [reply modules/comment/comment.permissions.js:50]: What is the purpose of rule hasRoles([ADMIN, SUPERADMIN, USER]) here if it always returns true? Mb you don't need to change this line?

File: modules/comment/comment.permissions.js [hunk 2]
```diff
@@ -47,8 +47,8 @@ const commentPermissionsMutations = {
...
+  deleteReplyForComment: or(isTheSameUser, hasRoles([ADMIN, SUPERADMIN, USER])),
```
Comment (AndriyLymych) [inline modules/comment/comment.permissions.js:51]: Actually, I don't a problem with the previous code 😃

Comment (ostgit12) [reply modules/comment/comment.permissions.js:51]: I checked it once more and it is working correctly. If we do not write role USER then the user can't delete his own comment.

File: modules/comment/comment.permissions.js (hunk 1 above)
Comment (SirAndrii) [reply modules/comment/comment.permissions.js:50]: So, what about `or( and(isTheSameUser, hasRoles[USER]),  hasRoles([ADMIN, SUPERADMIN]))` ?

Comment (YuriiDubnytskyi) [reply modules/comment/comment.permissions.js:50]: It was fixed on the frontend part and no need for changes on the backend. 
You could close this PR.

**Pull request closed without merging**